ontology/migrate_concepts_table.js

```diff
@@ -26,11 +26,11 @@ function transform_tbta_data(tbta_db) {
 						Roots							AS stem,
 						''								AS sense,
 						'${part_of_speech}'		AS part_of_speech,
-						Occurrences					AS occurrences,
+						0					AS occurrences,
```

Comment:
```suggestion
						0								AS occurrences,
```

Reply:
The spacing is all over the place for me, so I didn't bother touching it.
![image](https://github.com/user-attachments/assets/86e7ba45-ea1a-4c9b-bfde-a2b42d37cb63)